src/overlays/gamestates/ovl_select/z_select.c

```diff
@@ -588,34 +730,34 @@ void Select_Init(GameState* thisx) {
     this->state.main = Select_Main;
     this->state.destroy = Select_Destroy;
     this->scenes = sScenes;
-    this->unk_20C = 0;
+    this->topDisplayedScene = 0;
     this->currentScene = 0;
-    this->unk_1E0[0] = 0;
-    this->unk_1E0[1] = 0x13;
-    this->unk_1E0[2] = 0x25;
-    this->unk_1E0[3] = 0x33;
-    this->unk_1E0[4] = 0x3B;
-    this->unk_1E0[5] = 0x49;
-    this->unk_1E0[6] = 0x5B;
-    this->unk_1DC = 0;
+    this->pageDownStops[0] = 0;  // " 1:SPOT00"
+    this->pageDownStops[1] = 19; // "20:\x8Dﾄｷﾉﾏ"
+    this->pageDownStops[2] = 37; // "38:\x8Dﾀｶﾗﾊﾞｺﾔ"
+    this->pageDownStops[3] = 51; // "52:\x8Dﾊｶﾓﾘﾉｲｴ"
+    this->pageDownStops[4] = 59; // "60:\x8Cｿﾞｰﾗ\x8Dﾉﾐｾ"
+    this->pageDownStops[5] = 73; // "74:\x8Dｲﾄﾞｼﾀ \x8Cﾀﾞﾝｼﾞｮﾝ"
+    this->pageDownStops[6] = 91; // "92:\x8Cｶﾞﾉﾝ\x8Dﾉﾄｳ ｿﾉｺﾞ 3"
```

Comment:
I don't think these comments are of much use

Reply:
i think if the comments said the "useful" place name without the index it could be of use, but with the japanese and control codes it doesnt add any more helpful information

as a side note i would possibly suggest `pageLimit` as a name for pageDownStops, and call the index into it the `pageIndex`

Reply:
so yea my suggestion is 
```
this->pageDownStops[3] = 51; // Gravekeeper's Hut
this->pageDownStops[4] = 59; // Goron Shop
```

Reply:
`pageLimit` would be wrong, because it isn't limiting anything, it is jumping to one of those positions each time you press L

Reply:
those are the limits of each page to me, marking the page boundaries

Reply:
The differences aren't even, though, it's something like 19,18,14,8,14,18

Reply:
i dont think that fact is really relevant to the name "limit". to me its basically a synonym to "page down stops", the limit of which the page is finished
maybe there is a better word to describe my meaning, but i think in general we can improve pagedownstops